rust/Dockerfile

```diff
@@ -134,7 +134,8 @@ RUN set -xe \
 FROM development AS debug
 
 ARG TARGET
-COPY --from=builder /build/target/${TARGET}/debug/${PACKAGE} .
+## Build first with `cross build --target ${TARGET} -p ${PACKAGE} --debug && mv /target/${TARGET}/release/${PACKAGE} .`
```

Comment:
I don't understand the comment here. Is this saying what the Dockerfile did, or what the reader should do to replicate it manually without Docker?

Reply:
It's what the reader should do to get the image to package when building locally to test / debug release builds.

Reply:
Oh right, for running docker-compose with local builds.